src/renderer/components/ContextMenu/ContextMenuItem.js

```diff
@@ -28,7 +27,7 @@ class ContextMenuItem extends PureComponent<Props, State> {
     if (this.ref) {
       if (leftClick) {
         this.ref.addEventListener("click", this.showContextMenu);
-      } else if (!DISABLE_CONTEXT_MENU) {
```

Comment:
I wholeheartedly disagree we should drop this. Without this, inspecting an element that also has a context menu is a pita because of the menu and overlay.

Reply:
ok will rollback